### PR TITLE
Upload Images Component

### LINKED ISSUE #98: [TODO] Add Component to allow uploading images for the event
Create a component that allows the user to upload the Images. This component should be used for getting the user images for "Event Poster Image", "Event Organizer logo", and "Speakers picture".

image

It should have an image crop feature so that it fits the UI for showing the Event Poster, or any other place the crop size should be passed as a prop to the component.

https://user-images.githubusercontent.com/61188295/215274973-00732994-ac0d-414f-b0a9-67805d8fd848.mp4

Please feel free to comment and ask questions, if any. Thank you!

### PR DESCRIPTION
**Pull request by imf-ali**
This PR does:

- Able to add event header image
- Able to add event logo image
- Able to crop images

[screen-capture (3).webm](https://user-images.githubusercontent.com/42451210/224106357-4d83cb00-f1f2-41b4-9fa5-2f9ecbd29ef6.webm)

Fix #98

### CODE REVIEW

**Comment by imf-ali** [comment]
Hey @Dnouv, please have a look at the PR.

**Review by Dnouv — CHANGES_REQUESTED** [review]

File: app/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@apollo/client": "^3.5.9",
     "@jitsi/react-sdk": "^1.0.0",
     "@jitsi/web-sdk": "^0.2.0",
+    "@material-ui/core": "^4.12.4",
```
[the remark this answers is not in the document]
Comment (imf-ali) [reply app/package.json]: Fixed

**Comment by imf-ali** [comment]
Hey @Dnouv, I have done the required changes. Please have a look.

**Comment by imf-ali** [comment]
@Dnouv can you have a look at it?

**Pull request merged**